releasenotes/notes/feature-trotter-qrte-f7b28c4fd4b361d2.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Added Trotterization-based Quantum Real Time Evolution Algorithm
```

Comment:
It would be nice if the release note would contain a minimal example. But this can also wait until the rest of the reviews are done

Reply:
Added. I made it minimal (relying on default values). Let me know if you think we should flash out all arguments.